listing.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
```

Comment:
So that one should probably be shipped by a package, shared with the tripleo-validations one in order to get a common base of libs, right? @strider any thoughts?

Reply:
I totally agree! It's preliminary to provide basic listing and filtering for the container-based runs.